.github/workflows/logscribe.test.yaml

```diff
@@ -58,9 +59,16 @@ jobs:
         with:
           repo-token: ${{ secrets.GITHUB_TOKEN }}
 
-      - name: Run test
+      - name: Run logscribe tests
         run: |
           cd ./node
           task local:test-logscribe
         env:
           DATABASE_URL: "postgresql://postgres:postgres@localhost:5432/orakl-test?search_path=public"
+
+      - name: Run logscribeconsumer tests
+        run: |
+          cd ./node
+          task local:test-logscribeconsumer
+        env:
+          DATABASE_URL: "postgresql://postgres:postgres@localhost:5432/orakl-test?search_path=public"
```

Comment:
rather than having another step inside github action, it'd be better to just add another test in `run`
```
run: |
          cd ./node
          task local:test-logscrib
          task local:test-logscribeconsumer
```